.github/workflows/pull_request.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
       - name: Upload results using action from ${{ matrix.target }}
         env:
-          TRUNK_API_ADDRESS: https://api.trunk-staging.io/v1/metrics/createBundleUpload
+          TRUNK_API_ADDRESS: https://api.trunk-staging.io
```

Comment:
We'll need to update our own repo to match once this is live.

Reply:
should we still call this TRUNK_API_ADDRESS? maybe TRUNK_API_ORIGIN would be more appropriate?

Reply:
`API_ADDRESS` is a pattern we use elsewhere.